mis_builder/models/aep.py

```diff
@@ -421,6 +425,10 @@ def f(mo):
                                      (AccountingNone, AccountingNone))
             if field == 'bal':
                 v = debit - credit
+            elif field == 'pbal' and debit >= credit:
+                v += debit - credit
```

Comment:
Here (and below) should be assignation or we get:
```
UnboundLocalError: local variable 'v' referenced before assignment
```

Reply:
That was a copy-paste error indeed.